unlock-js/src/v01/getLock.js

```diff
@@ -48,6 +49,9 @@ export default function(address) {
 
   // Once all lock attributes have been fetched
   return Promise.all(constantPromises).then(() => {
+    // numberOfOwners was previously called outstandingKeys. In order to keep compatibility
+    // we also assign it. This behavior will eventually be deprecated
+    update.outstandingKeys = update.numberOfOwners
```

Comment:
We renamed outstanding keys to `totalSupply` to follow the ERC721 spec.  `numberOfOwners` may be slightly different

Reply:
Ho, that is a good point. I'll update that! Thanks.

Reply:
Done ;)